docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -139,6 +139,11 @@ This change only needs to be done on the first server of the cluster. It will be
 
 In `conf/pf.conf` :
 
+----
+[general]
+hostname=pf
```

Comment:
Why do we need to do this ?

Reply:
By default, for a cluster, it will take the first cluster member hostname.
(See in deployement with OeufDur)

Reply:
By default its 'packetfence', the hostname of the server shouldn't have any impact on this setting.

I'll check with ~~@febreeze~~ @fdurand